cds/webapi/C#/MetadataOperations/CDSWebApiServiceExtensions.cs

```diff
@@ -0,0 +1,2312 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PowerApps.Samples.Metadata
+{
+    public static class CDSWebApiServiceExtensions
+    {
+        /// <summary>
+        /// The strong consistency header.
+        /// Strong consistency offers a linearizability guarantee. Linearizability refers to serving requests concurrently.
+        /// The reads are guaranteed to return the most recent committed version of an item.
+        /// Users are always guaranteed to read the latest committed write.
+        /// Use of this header incurs a small overhead and is therefore not suited to high volume workloads.
+        /// </summary>
+        private static readonly Dictionary<string, List<string>> STRONGCONSISTENCYHEADER = new Dictionary<string, List<string>>()
```

Comment:
Can we name it strongConsistencyHeader?

 private static readonly Dictionary<string, List<string>> strongConsistencyHeader = new Dictionary<string, List<string>>()

Reply:
Done